### ISSUE TITLE
first run on new install failing

### ISSUE THREAD

**Issue by Ornias1993** [jailmanager/jailman#85]
First run on a fresh freenas install seems to be failing due to iocage init.
Should be catched, but if not it should be fixed by `set -e`

It gives the following error:
```
cannot open 'Setting': dataset does not exist
cannot open 'up': dataset does not exist
cannot open 'zpool': dataset does not exist
cannot open '[nameofzpool]': invalid character '[' in name
cannot open 'for': dataset does not exist
cannot open 'iocage': dataset does not exist
cannot open 'usage': dataset does not exist
cannot open 'If': dataset does not exist
cannot open 'you': dataset does not exist
cannot open 'wish': dataset does not exist
cannot open 'to': dataset does not exist
cannot open 'change': dataset does not exist
cannot open 'please': dataset does not exist
cannot open 'use': dataset does not exist
cannot open '"iocage': invalid character '"' in name
cannot open 'activate"': invalid character '"' in name
cannot open 'Creating': dataset does not exist
cannot open 'Creating': dataset does not exist
cannot open 'Creating': dataset does not exist
cannot open 'Creating': dataset does not exist
cannot open 'Creating': dataset does not exist
cannot open 'Creating': dataset does not exist
cannot open 'Creating': dataset does not exist
cannot open 'Default': dataset does not exist
cannot open 'configuration': dataset does not exist
cannot open 'missing,': invalid character ',' in name
cannot open 'creating': dataset does not exist
cannot open 'one': dataset does not exist
```

**Comment by Ornias1993** [comment]
it seems to pipe the first iocage output into commands.

**Comment by Ornias1993** [comment]
Due to the use of plugins, this might be solved in #200

**Comment by Ornias1993** [comment]
Fixed on the dev branch...
